app_customers/lib/pages/dashboard/dashboard_view.dart

```diff
@@ -125,6 +128,7 @@ class DashboardView extends StatelessWidget {
 class _DashboardBody extends StatelessWidget {
   final autoSizeGroup = AutoSizeGroup();
   final List<_DashboardPage> pages;
+  bool isTransforming = false;
```

Comment:
When should this variable be used?

Reply:
ohh... its was for an old program... i remove it